upgrade-2.x.sh

```diff
@@ -2,6 +2,7 @@
 
 # default configuration
 NOREBOOT=no
+DELTA_VERSION=3
```

Comment:
I've tested this back to the earliest multicontainer-capable OS version (2.11.0), but it'd be good to actually confirm these deltas can be applied before that. @robertgzr do you know when balena-engine deltas became fully supported in the OS?

Reply:
I don’t remember the exact version off the top of my head, I’ll check tomorrow, but they’re generally available since the OS version that switched to balenaEngine from docker.

Reply:
Okay, found it, the oldest available OS version with balenaEngine is v2.9.0: https://github.com/balena-os/meta-balena/blob/master/CHANGELOG.md#v290

Reply:
Awesome, thanks for confirming @dfunckt. I've confirmed a few different upgrade paths around that changeover, so looks good.